test/unit/specs/store/transactions.spec.js

```diff
@@ -112,15 +112,14 @@ describe(`Module: Transactions`, () => {
   })
 
   it(`should set error to true if dispatches fail`, async () => {
-    let err = `unexpected error`
+    let err = new Error(`unexpected error`)
     const dispatch = jest.fn(() => {
-      store.commit(`setError`, err)
-      throw new Error(err)
+      store.state.error = err.message.slice(0)
```

Comment:
this sets the error in the dispatch, but the error should be set in response to the dispatch throwing an error.